### PR TITLE
Update URLs to use https instead of http

### PR DESCRIPTION
**Pull request by pabs3**
Suggested-by: check-all-the-things

### CODE REVIEW

**Review by gnikonorov — APPROVED** [review]

**Comment by gnikonorov** [comment]
bump for a merge from @sallner and/or @icemac

**Review by hugovk — APPROVED** [review]
Confirmed new links work. Thanks!

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ Preparing Pull Requests
 #. Install `tox <https://tox.readthedocs.io/en/latest/>`_::
 
    Tox is used to run all the tests and will automatically setup virtualenvs
-   to run the tests in. Implicitly http://www.virtualenv.org/en/latest/ is used::
+   to run the tests in. Implicitly https://virtualenv.pypa.io/en/latest/ is used::
```
Comment (icemac) [inline CONTRIBUTING.rst:33]:
```suggestion
   to run the tests in. Implicitly https://virtualenv.pypa.io/ is used::
```

**Review by icemac — APPROVED** [review]
LGTM besides one simplification.

**Pull request merged**